lib/node_modules/@stdlib/math/base/complex/divide/docs/repl.txt

```diff
@@ -0,0 +1,40 @@
+
+{{alias}}( [out,] re1, im1, re2, im2 )
+    Divides two complex numbers.
+
+    Parameters
+    ----------
+    out: Array|TypedArray|Object
```

Comment:
Would https://github.com/stdlib-js/stdlib/blob/develop/docs/style-docs be a good place for the repl guide since it's mostly formatting information? I looked for a guide but missed it.